### PR TITLE
Header size reduced when outside landing page.

### LINKED ISSUE #52: Shrink version of the header menu when outside the landing page
Can we shrink the header  it as much as possible when viewing pages that are not the landing page?

### PR DESCRIPTION
**Pull request by deveshidwivedi**
**GitHub Issue:** #52
Resolves #52 

**Summary**:
The header now is reduced in size for all the pages except the landing page, as mentioned in the issue. 

**Do you think resolving this issue might require an [Architectural Decision Record (ADR)](https://github.com/json-schema-org/community/blob/main/CONTRIBUTING.md#key-architectural-decisions)? (significant or noteworthy)**

No

### CODE REVIEW

**Comment by deveshidwivedi** [comment]
Please let me know if other changes are required, thank you!

File: components/Layout.tsx [hunk 1]
```diff
@@ -52,7 +58,7 @@ export default function Layout({ children, mainClassName, metaTitle, whiteBg }:
         <main className={
           classnames(mainClassName, 'z-10 xl:rounded-xl py-4 mx-auto')
         }>
-          <header className={classnames('w-full bg-white fixed top-0 z-[170] shadow-xl drop-shadow-lg')}>
+          <header className={classnames('w-full bg-white fixed top-0 z-[170] shadow-xl',  { 'h-16': !isLandingPage, 'drop-shadow-lg': isLandingPage, 'shadow-sm': !isLandingPage })}>
```
Comment (benjagm) [inline components/Layout.tsx:61]: Screenshot 2024-02-23 at 02 05 41

This is what I see. I think we need to reduce the space on top and try to center the elements vertically to have the same space between the top and the bottom. We can also reduce the size of the logo.

We should check the mobile versión too.

**Review by benjagm — CHANGES_REQUESTED** [review]
Thanks a lot for this PR!

This was a first good step, however I think we need some more adjustments to produce the best result.

**Review by benjagm — CHANGES_REQUESTED** [review]
Screenshot 2024-02-23 at 11 37 37

Looks better! However we have now a lot of unused space between the header and the sidebar and the content. We need to move the content upper.

**Comment by deveshidwivedi** [comment]
hi @benjagm! does this look okay?

**Comment by benjagm** [comment]
> hi @benjagm! does this look okay?

I don't see any change. We need to decrease that huge space between the header and the content.

**Comment by benjagm** [comment]
@deveshidwivedi let me know if you need help with this one!

**Comment by deveshidwivedi** [comment]
Sorry for the delay @benjagm. I have reduced the padding on top and margin on bottom (to make it look even), please let me know if it can get better, thanks.

**Comment by benjagm** [comment]
> I have reduced the padding on top and margin on bottom (to make it look even), please let me know if it can get better, thanks.

You will need to move the sidebar up as well. (You only moved the page's body)

**Review by benjagm — CHANGES_REQUESTED** [review]
You will need to move the sidebar up as well. (You only moved the page's body)

**Review by benjagm — APPROVED** [review]
Nice job @deveshidwivedi!! Looks great!

Can you take a look to the conflict in components/Layout.tsx? Probably many changes since you created the fork.

Thanks a lot!

**Comment by deveshidwivedi** [comment]
done!

**Comment by benjagm** [comment]
Thanks @deveshidwivedi !! When doing the last test before merging I found something we need to fix in mobile.
This is what we have in this PR:
Screenshot 2024-03-01 at 10 31 34

But this is what we should have:
Screenshot 2024-03-01 at 10 31 50

We need to do the correction in mobile too to move the content up.

**Comment by deveshidwivedi** [comment]
updated!

**Comment by benjagm** [comment]
Thanks @deveshidwivedi Can you please sqash and rebase ? We'll merge this.

**Comment by deveshidwivedi** [comment]
I think I'll open another PR for this @benjagm. I'm sorry!

**Pull request closed without merging**